app/src/main/java/com/mionix/myapplication/DB/DataDao.kt

```diff
@@ -10,12 +11,17 @@ interface DataDao {
     @Insert
     fun saveData(data : DataTable)
     @Query("select * from DataTable WHERE dataID = :dataID")
-    fun readData(dataID : Int) :DataTable
+    fun readData(dataID : Int) :DataTable?
     @Query("select * from DataTable")
     fun readAllData() :List<DataTable>
+    @Query("select * from DataTable LIMIT 25 OFFSET :startAt")
+    fun readMoreData(startAt:Int):MutableList<DataTable>
```

Comment:
Tại sao trả vể MutableList?

Reply:
cho nó giống với kiểu dữ liệu với cái list để bỏ vào adapter

Reply:
> Reply
Rồi có sửa cái gì trong list đó không? nếu không thì hoàn toàn có thể set 1 list vào 1 biên mutablelist